package.json

```diff
@@ -43,6 +47,7 @@
     "test": "mocha --reporter spec --bail --check-leaks test/",
     "test-ci": "nyc --reporter=lcov --reporter=text npm test",
     "test-cov": "nyc --reporter=text npm test",
+    "test-types": "npx -p typescript tsc --project tsconfig.json --noEmit",
```

Comment:
Run the typescript checker via `npx`. `typescript` package cannot be listed in  `devDependencies` due to support of node@0.8 which is incompatible.